torch/csrc/jit/fusion_compiler.cpp

```diff
@@ -780,3 +781,72 @@ FusionCompiler & sharedFusionCompiler() {
 }
 
 }}
+
+# else
```

Comment:
Fuser is disabled on Windows because the current implementation uses symbols from Linux-specific headers `dlfcn.h` and `unistd.h`. We will need to find alternatives for Windows.